packages/backend/server/src/plugins/payment/manager/user.ts

```
@@ -683,4 +684,17 @@ export class UserSubscriptionManager extends SubscriptionManager {
     }
   }
+
+  @OnEvent('user.deleted')
+  async onUserDeleted({ id }: Events['user.deleted']) {
+    const subscription = await this.db.subscription.findFirst({
+      where: {
+        targetId: id,
+      },
+    });
+
+    if (subscription && subscription.stripeSubscriptionId) {
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)